docs/getting_started/introduction.md

```diff
@@ -0,0 +1,71 @@
+
+
+# What is Time-Series Forecasting?
+
+Generally speaking, forecasting just means making predictions about events in
+the feature. Trivially, while weather forecasting tries to predict the
```

Comment:
If one encode the weather (sunny, cloudy, rainy) into numbers and we it is also a forecasting problem right? Maybe demand forecasting is a more common example?

Reply:
The idea was to name something which isn't time-series forecasting.